ledger.sql

```diff
@@ -0,0 +1,12 @@
+CREATE  TABLE ledger
+(
+        PRIMARY KEY (ledger_id),
+
+        ledger_id CHAR(10) NOT NULL,
+			CONSTRAINT ledger_id_not_blank
+			CHECK (ledger_id <> '')
+
+        description VARCHAR(50) NULL
```

Comment:
I'd recommend making this NOT NULL and having a constraint that doesn't let it be blanks.